### PR TITLE
[Fix #4910] added example to space inside block braces

### PR DESCRIPTION
**Pull request by yieldjessyield**
Added an example documentation to `space_inside_block_braces.rb` cop.

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [ ] Added tests.
* [ ] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] All tests(`rake spec`) are passing.
* [x] The new code doesn't generate RuboCop offenses that are checked by `rake internal_investigation`.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: lib/rubocop/cop/layout/space_inside_block_braces.rb [hunk 1]
```diff
@@ -7,6 +7,15 @@ module Layout
       # them on configuration. For blocks taking parameters, it checks that the
       # left brace has or doesn't have trailing space depending on
       # configuration.
+      #
+      # @example
+      #   # bad
```
Comment (jonatas) [inline lib/rubocop/cop/layout/space_inside_block_braces.rb]: `# bad` if `EnforcedStyleForEmptyBraces` config is set with `space`, right?

Maybe you enhance it and add also the inverse configuration.

Comment (pocke) [reply lib/rubocop/cop/layout/space_inside_block_braces.rb]: For example: https://github.com/bbatsov/rubocop/blob/fe87238e769d00cb52212c640fdfaefc66699a1b/lib/rubocop/cop/style/alias.rb#L10-L29

**Comment by jonatas** [comment]
@yieldjessyield: Just a few reminders of good commit messages:

Use the [imperative](https://chris.beams.io/posts/git-commit/#imperative) mood in the subject line, and also `Capitalize` the subject message.

**Comment by bbatsov** [comment]
It seems someone already wrote the missing documentation for this cop. We ran into a classic race condition :-( 

@yieldjessyield Thanks for your contribution and I'm sorry we can't merge it. Excellent work! (P.S. There are plenty more undocumented cops)

**Pull request closed without merging**